lib/taskana-core/src/test/java/pro/taskana/impl/WorkbasketServiceImplTest.java

```diff
@@ -116,79 +115,79 @@ public void testCreateWorkbasket_WithDistibutionTargets()
         assertThat(actualWb.getModified(), not(equalTo(null)));
     }
 
-    @Test(expected = WorkbasketNotFoundException.class)
-    public void testCreateWorkbasket_DistibutionTargetNotExisting()
-        throws WorkbasketNotFoundException, NotAuthorizedException, InvalidWorkbasketException,
-        WorkbasketAlreadyExistException, DomainNotFoundException {
+    @Test
+    void testCreateWorkbasket_DistibutionTargetNotExisting()
+        throws NotAuthorizedException, WorkbasketNotFoundException {
         WorkbasketImpl expectedWb = createTestWorkbasket("ID-1", "Key-1");
         when(internalTaskanaEngineMock.domainExists(any())).thenReturn(true);
 
-        try {
-            cutSpy.createWorkbasket(expectedWb);
-            String id1 = "4711";
-            List<String> destinations = new ArrayList<>(Arrays.asList(id1));
-            cutSpy.setDistributionTargets(expectedWb.getId(), destinations);
-            doThrow(WorkbasketNotFoundException.class).when(cutSpy).getDistributionTargets(expectedWb.getId()).get(0);
-
-        } catch (WorkbasketNotFoundException e) {
-            verify(internalTaskanaEngineMock, times(3)).openConnection();
-            verify(workbasketMapperMock, times(1)).insert(expectedWb);
-            verify(workbasketMapperMock, times(1)).findById(any());
-            verify(workbasketMapperMock, times(1)).findByKeyAndDomain(any(), any());
-            verify(cutSpy, times(1)).getWorkbasket(any());
-            verify(internalTaskanaEngineMock, times(3)).returnConnection();
-            verify(taskanaEngine, times(2)).checkRoleMembership(any());
-            verify(internalTaskanaEngineMock, times(2)).getEngine();
-            verify(internalTaskanaEngineMock, times(1)).domainExists(any());
-            verifyNoMoreInteractions(taskQueryMock, taskServiceMock, workbasketMapperMock, workbasketAccessMapperMock,
-                distributionTargetMapperMock,
-                internalTaskanaEngineMock, taskanaEngine, taskanaEngineConfigurationMock);
-            throw e;
-        }
+        WorkbasketNotFoundException e = Assertions.assertThrows(
+            WorkbasketNotFoundException.class, () -> {
```

Comment:
is this assertThrows really necessary?

Reply:
what is this test doing? From my point of view this one is useless and tests nothing ...